src/main/java/com/example/pupbuddy/dto/Chore.java

```diff
@@ -4,7 +4,7 @@
 
 @Data
 public class Chore {
-    private int ChoreId;
+    private int choreId;
```

Comment:
lowerCamelCase is correct.
Since Chore is given by the class name, I'd remove that from the variable name.  It's redundant.